scripts/clean-db-dump.sql

```diff
@@ -0,0 +1,102 @@
+\echo 'Update users'
+UPDATE users
+SET name = 'Test user',
+    email_address = id || '@user.marketplace.team',
+    password = :bcrypt_password;
+
+-- Remove data about currently ongoing framework applications
+\echo 'Delete draft services for open frameworks'
+DELETE
+FROM draft_services
+WHERE framework_id IN
+    (SELECT id
+     FROM frameworks
+     WHERE status='open');
+
+\echo 'Delete supplier frameworks for open frameworks'
+DELETE
+FROM supplier_frameworks
+WHERE framework_id IN
+    (SELECT id
+     FROM frameworks
+     WHERE status='open');
+
+-- Remove data related to open DOS procurements
+-- (We can't tell if a procurement is ongoing, so delete all brief responses)
+\echo 'Delete brief responses'
+DELETE
+FROM brief_responses;
+
+\echo 'Delete draft briefs'
+\echo '  > Delete draft brief users'
+DELETE
+FROM brief_users
+WHERE brief_id IN
+    (SELECT id
+     FROM briefs
+     WHERE published_at IS NULL);
+
+\echo '  > Delete draft briefs'
+DELETE
+FROM briefs
+WHERE published_at IS NULL;
+
```

Comment:
This could be a little clearer if it was defined as some plpgsql functions/ variables but I gather this is mostly a copy paste from another script right? In that case probably out of scope to rewrite all this?